MechanicsCore/src/main/java/me/deecaad/core/mechanics/defaultmechanics/MythicSkillMechanic.java

```diff
@@ -50,22 +52,26 @@ protected void use0(CastData cast) {
         Collection<Entity> eTargets = new ArrayList<>();
         Collection<Location> lTargets = new ArrayList<>();
 
-        OUTER : for (Iterator<CastData> it = targeter.getTargets(cast); it.hasNext();) {
+        OUTER:
+        for (Iterator<CastData> it = targeter.getTargets(cast); it.hasNext(); ) {
             CastData target = it.next();
             for (Condition condition : conditions)
                 if (!condition.isAllowed(target))
                     continue OUTER;
 
-            if (target.getTarget() != null)
+            if (target.getTarget() != null) {
                 eTargets.add(target.getTarget());
-            else
+                target.getTarget().setMetadata("doing-skill-damage", new LazyMetadataValue(MechanicsCore.getPlugin(), target::getTarget));
```

Comment:
This definitely has side effects... This is a hook we use that we expect MythicMobs to provide... We should never be setting it ourselves. Also, we never clean this metadata so that tag might just sit there.

Reply:
Ah that's why I couldn't find it in the code. Maybe mythicmobs has changed this. I'll have a look.

Reply:
Found it, will do some cleanup and send another pull request.